eagle-external/hadoop_jmx_collector/config-sample.json

```diff
@@ -12,18 +12,35 @@
        "https":false
      }
    },
-   "input": {
-    "component": "namenode",
-    "port": "50070",
-    "https": false
-   },
+   "inputs": [
+      {
+        "component": "namenode",
+        "host": "127.0.0.1",
+        "port": "50070",
+        "https": false,
+        "kafka_topic": "nn_jmx_metric_sandbox"
+      },
+      {
+        "component": "resourcemanager",
+        "host": "127.0.0.1",
+        "port": "8088",
+        "https": false,
+        "kafka_topic": "rm_jmx_metric_sandbox"
+      },
+      {
+        "component": "datanode",
+        "host": "127.0.0.1",
```

Comment:
let's use place holder like what we do in PR312, use server.eagle.apache.org. So that developer could simply use /etc/resolv.conf when there is different endpoint he want to change (etc, hdp sandbox)